internal/v1/handler_blueprints_test.go

```diff
@@ -837,16 +864,124 @@ func TestHandlers_GetBlueprint(t *testing.T) {
 	}
 }
 
+func marshalAndUnmarshalPrint(t *testing.T, v interface{}) []byte {
+	data, err := json.Marshal(v)
+	require.NoError(t, err)
+
+	var dataUnmarshal interface{}
+	err = json.Unmarshal(data, &dataUnmarshal)
+	require.NoError(t, err)
+
+	data, err = json.MarshalIndent(dataUnmarshal, "", "  ")
+	require.NoError(t, err)
+
+	return data
+}
+
+func compareOutputExportBlueprint(t *testing.T, jsonResponse []byte) {
+	file, err := os.CreateTemp(t.TempDir(), "exported_blueprint_temp.*.json")
+	require.NoError(t, err)
+
+	jsonFile, err := os.Create(file.Name())
+	require.NoError(t, err)
+
+	_, err = jsonFile.Write(jsonResponse)
+	require.NoError(t, err)
```

Comment:
There is no need to create a temporary file, Go's `Reader` and `Writer` interface is really powerful. Just write it into a bytes buffer or string, then you can use the same string in the `JSONEq` function below.